fabfile/run.py

```diff
@@ -1,6 +1,12 @@
-from fabric.api import task, run, cd, runs_once, roles
+from fabric.api import task, run, cd, runs_once, roles, execute
 
 import config
+import use
+import prefix
+import pg
+import setup
+import utils
+import re
```

Comment:
are all of these necessary ?

Reply:
Do you mean some of the imports may import other ones recursively ?

Reply:
I meant there are many imports here.  Are all of them necessary

Reply:
Yes they are.